script/cfg.py

```diff
@@ -233,7 +233,7 @@ def pre_openqa_call_start(repos):
         ''' + (repo0folder if repo0folder else "") + '''
         [ "$arch" != . ] || arch=x86_64
         ''' + openqa_call_news(news, news_archs) + '''
-        echo "/usr/share/openqa/script/client isos post --host localhost \\\\\"
+        echo "/usr/bin/openqa-cli -X post isos --host localhost \\\\\"
```

Comment:
seems `api` missed here.
should be `openqa-cli api -X post`. Following lines are same.

Reply:
thx, fixed